src/GitHub.VisualStudio/Helpers/ActiveDocumentSnapshot.cs

```diff
@@ -19,7 +19,7 @@ public class ActiveDocumentSnapshot : IActiveDocumentSnapshot
         public ActiveDocumentSnapshot([Import(typeof(SVsServiceProvider))] IServiceProvider serviceProvider)
         {
             StartLine = EndLine = -1;
-            Name = Services.Dte2?.ActiveDocument?.FullName;
+            Name = Services.Dte2?.ActiveDocument?.ProjectItem.Name;
```

Comment:
`ProjectItem.ContainingProject` references the project where the file is, and turns out to be null when testing it out. However, debugging `Services.Dte2.Solution.Projects` contains 1 item, which I assume is the solution.

It's possible also to find a file through `Services.Dte2.Solution.FindProjectItem("somefile")` which might mean that any file, whether it be in solution or project, is considered a `ProjectItem`